### ISSUE TITLE
Params needs to be reset after opening each file

### ISSUE THREAD

**Issue by MathiasGilson** [abreheret/PixelAnnotationTool#18]
Defining the alpha mask, resize factor and circle size params is not persistent after opening another file

Expected behavior:

Set the params once and have them applied by default when opening a new file

**Comment by quickeye** [comment]
I agree with Mathias as keeping alpha, resize, and circle size parameters between images.

Adding the following shortcuts would also help:
Page Up + Page Down would go to the prev / next image. (This feature exists in other labeling programs)

Add a hotkey to switch back to the previous class. I use 2 classes for foreground / background, and switching back and forth with a hotkey would be much faster.

**Comment by BestVanRome** [comment]
> Adding the following shortcuts would also help:
> Page Up + Page Down would go to the prev / next image. (This feature exists in other labeling programs)

This is possible through pushing **Ctrl+Shift+Page up/Page down**
